### ISSUE TITLE
Actuals Balance Sheet Reporting - Consolidated

### ISSUE THREAD

**Issue by afs1jes** [afs1jes/Clarins-US-IR-Migration#2]

**Comment by janeslee25** [comment]
I get an error when I run template.


![Image](https://user-images.githubusercontent.com/124628964/220988306-495d7a66-5bc8-4b7c-8932-5a2857dbffad.png)

**Comment by zgrose** [comment]
You probably skipped one of the selections? You have to make a selection on every dialog. If you Finish early, this can happen.

**Comment by zgrose** [comment]
What's the issue identified, @janeslee25 ?

**Comment by janeslee25** [comment]
I formatted the numbers to have no decimals - format defaults to 2 decimal places when i run

**Comment by zgrose** [comment]
where did you apply the formatting? if you tried to apply to the first row in the grid area, that's going to be overwritten by the format types above.

**Comment by zgrose** [comment]
Also, it's pretty much meaningless to have Format using styles **and** use the format types (leaf/rollup/grand/type1). I suspect that is what's going on here? @janeslee25

**Comment by janeslee25** [comment]
Yes -you are correct. I just adjusted the formatting in the correct place - but it is still defaulting to the cube default.

**Comment by zgrose** [comment]
Maybe you're looking at a different template? I just applied no decimal formatting to the format rows and they applied as I would expect:

<img width="1228" alt="Screenshot 2023-03-20 163929" src="https://user-images.githubusercontent.com/5044847/226471774-1971d92d-4da6-43f8-a078-b5b9a4d1768e.png">

**Comment by zgrose** [comment]
The model built at this link: https://clarins-usdev.a3solutions.com/Portal/customize/5b146f36-4bd8-4d4d-ab51-7049ea212190

**Comment by zgrose** [comment]
@janeslee25 just a ping in case you only get notifications that way. if you're seeing the problem persist, let's have a quick screenshare at your next available opportunity.

**Comment by janeslee25** [comment]
I just reran. It works. So does this mean I have to run through the website in order for me to see the change. I was running it in Excel only. And it would return numbers with decimals. I even closed and reopened and ran.

**Comment by zgrose** [comment]
@janeslee25 dunno what you were doing differently than me, but I just changed the formatting in the formatting rows and then saved. your last file saved to the server had 0.00 in the formatting rows.
